### ISSUE TITLE
Calendar highlights won't be removed after attribute changes

### ISSUE THREAD

**Issue by nikwins** [nathanreyes/v-calendar#312]
Since Beta 7/8 there is a bug that when I e.g. add some attributes to the calendar to show a specific date range and then remove them again later, the `vc-highlight` containers won't be removed although the attribute to build them isn't attached to the calendar anymore. So the highlights will still be shown, when they should've been removed.

UPDATE: I found that setting the `locale` property causes this bug. [Here](https://codesandbox.io/s/p5jwvw3lk7?fontsize=14) is an example. If you remove the `locale` property it works fine.

**Comment by nathanreyes** [comment]
So the quick fix is storing a `locale` variable in your data instead of putting it inline in the template.
```html
<v-date-picker
  mode="range"
  :value="null"
  :attributes="attributes"
  :locale="locale"
  is-dark
  />
```
```js
  ...
  data() {
    return {
      locale: { id: 'de', firstDayOfWeek: 2, masks: { weekdays: 'WW' } },
      attributes: [],
    };
  }
  ...
```
What is happening is that the `locale` prop is getting reevaluated any time the `v-date-picker` component is re-rendered. The component is getting confused (which shouldn't happen) when both the `locale` and `attribute` props are changing at the same time.

So I'll mark this as a bug, but re-working it as described above should get you going.

**Comment by nikwins** [comment]
Great, thanks!

**Comment by nathanreyes** [comment]
Fixed in `1.0.0-beta.12`
